autoload/hacx.wad/brghtmps.lmp

```diff
@@ -0,0 +1,82 @@
+BRIGHTMAP NOTGRAYORBROWN 4,9-63,112-125,152-255
+BRIGHTMAP REDONLY 45,173-191
+BRIGHTMAP GREENONLY1 112-127
+BRIGHTMAP REDANDGREEN 16-47,112-127,173-191
+BRIGHTMAP CHEXRED 176-177
+BRIGHTMAP HACXLIGHTNING 4,168,192-199,208,224
+
+// {"BFALL1",   DOOM2ONLY, redandgreen},
+// {"BFALL2",   DOOM2ONLY, redandgreen},
+// {"BFALL3",   DOOM2ONLY, redandgreen},
+// {"BFALL4",   DOOM2ONLY, redandgreen},
+TEXTURE BRNSMALR GREENONLY1
+TEXTURE DOORRED REDANDGREEN
+TEXTURE SLADWALL CHEXRED
+// {"SW1BRCOM", DOOM2ONLY, redonly},
+// {"SW1BRN1",  DOOM2ONLY, redandgreen},
+TEXTURE SW1BRN2  NOTGRAYORBROWN
+TEXTURE SW1BRNGN NOTGRAYORBROWN
+// {"SW1BROWN", DOOM2ONLY, notgrayorbrown},
+// {"SW2BRCOM", DOOM2ONLY, greenonly1},
+// {"SW2BRN1",  DOOM2ONLY, redandgreen},
+TEXTURE SW2BRN2 NOTGRAYORBROWN
+// {"SW2BROWN", DOOM2ONLY, notgrayorbrown},
+TEXTURE COMPSPAN GREENONLY1
+TEXTURE COMPSTA1 NOTGRAYORBROWN
+// {"COMPSTA2", DOOM2ONLY, notgrayorbrown},
+TEXTURE HD5 REDANDGREEN
+// {"HD8",      DOOM2ONLY, redandgreen},
+// {"HD9",      DOOM2ONLY, redandgreen},
+TEXTURE BLAKWAL2 REDANDGREEN
+TEXTURE CEMENT7  GREENONLY1
+TEXTURE ROCK4    REDONLY
+// {"SLOPPY1",  DOOM2ONLY, notgrayorbrown},
+// {"SPCDOOR4", DOOM2ONLY, notgrayorbrown},
+TEXTURE ZZZFACE1 GREENONLY1
+TEXTURE ZZZFACE2 REDANDGREEN
+TEXTURE HW166    REDANDGREEN
+TEXTURE HW510    NOTGRAYORBROWN
+TEXTURE HW511    NOTGRAYORBROWN
+TEXTURE HW512    NOTGRAYORBROWN
+
+// Chainsaw
+SPRITE CSAW REDONLY
+// Plasmagun
+SPRITE PLAS REDONLY
+// Cell Charge
+SPRITE CELL REDONLY
+// Cell Charge Pack
+SPRITE CELP REDONLY
+// Rocket launcher
+SPRITE LAUN REDANDGREEN
+// Medikit
+SPRITE MEDI REDANDGREEN
+// Rocket
+SPRITE ROCK GREENONLY1
+// Box of rockets
+SPRITE BROK GREENONLY1
+// Health Bonus
+SPRITE BON1 NOTGRAYORBROWN
+// Stimpack
+SPRITE STIM NOTGRAYORBROWN
+
+FLAT FLOOR1_1 NOTGRAYORBROWN
+FLAT FLOOR1_7 NOTGRAYORBROWN
+FLAT FLOOR3_3 NOTGRAYORBROWN
+FLAT NUKAGE1 NOTGRAYORBROWN
+FLAT NUKAGE2 NOTGRAYORBROWN
+FLAT NUKAGE3 NOTGRAYORBROWN
+FLAT BLOOD1 GREENONLY1
+FLAT BLOOD2 GREENONLY1
+FLAT BLOOD3 GREENONLY1
+FLAT SLIME13 NOTGRAYORBROWN
+FLAT SLIME14 NOTGRAYORBROWN
+FLAT SLIME15 NOTGRAYORBROWN
+
+STATE 72 HACXLIGHTNING // S_SAW2
+STATE 73 HACXLIGHTNING // S_SAW3
```

Comment:
@JNechaevsky How to check if `HACXLIGHTNING` is working? I'm not sure. There are also a lot of commented texture brightmaps for Hacx, should we keep them?

Reply:
Just load this map along with hacx.wad, it have black room where you can see if b-map is working on saw (small red dots). Type `BLAST` to have all weapons: [blackmap.zip](https://github.com/fabiangreffrath/woof/files/10368853/blackmap.zip)

But I honestly can't remember, probably it wasn't me who wrote Hacx brightmaps. Was it @fabiangreffrath then? 🤔

Reply:
Yes I saw red dots on the saw but not sure about other things, never played Hacx before.

Reply:
> But I honestly can't remember, probably it wasn't me who wrote Hacx brightmaps. Was it @fabiangreffrath then? 🤔

Well, I commited them some five years ago, but at least you apparently gave some "suggestions". 😉 
https://github.com/fabiangreffrath/crispy-doom/commit/a37f8fc79950c9a26c298b5d53adbce8b00ac638

BTW, our brightmap test rooms can all be found here:
https://github.com/fabiangreffrath/test-wads

Reply:
> puts some love into REKKR

I think for a first shot it may be sufficient to highlight some blue and yellow pixels in the SW1BLUE and SW2BLUE textures. Also, some of the player sprites show potential to highlight some red (weapons 4, 6, 7) or gold (weapon 5) pixels.